packages/vue/src/components/organisms/SfAccordion/SfAccordion.js

```diff
@@ -4,5 +4,64 @@ import SfAccordionItem from "./_internal/SfAccordionItem.vue";
 Vue.component("SfAccordionItem", SfAccordionItem);
 
 export default {
-  name: "SfAccordion"
+  name: "SfAccordion",
+  props: {
+    /**
+     * Array of items to populate the default markup
+     */
+    items: {
+      type: Array,
+      default: () => []
+    },
+    /** Callback function to handle click on accordion item content element */
+    contentClickedCallback: {
```

Comment:
imo we don't need this prop, I think is better to just emit a `click` event, wdyt?